samcli/commands/init/interactive_init_flow.py

```diff
@@ -156,8 +160,12 @@ def _generate_from_app_template(
     Dependency Manager: {dependency_manager}
     Output Directory: {output_dir}
 
-    Next steps can be found in the README file at {output_dir}/{name}/README.md
-        """
+    Next application steps can be found in the README file at {output_dir}/{name}/README.md
+    
+    Commands you can use next
+    =========================
+    [*] Create pipeline: cd {name} && sam pipeline init --bootstrap
+    """
```

Comment:
Small UX nit, we usually print next steps in yellow do we want to follow same pattern here?

Reply:
Thanks updated to use yellow, screenshot in comments